lib/dialects/postgres/query-generator.js

```diff
@@ -535,7 +535,13 @@ class PostgresQueryGenerator extends AbstractQueryGenerator {
     }
 
     if (attribute.comment && _.isString(attribute.comment)) {
-      sql += ` COMMENT ${attribute.comment}`;
+      if (option && option.context === 'addColumn') {
+        const quotedAttr = this.quoteIdentifier(option.key);
+        const escapedCommentText = this.escape(attribute.comment);
+        sql += `; COMMENT ON COLUMN ${this.quoteTable(option.table)}.${quotedAttr} IS ${escapedCommentText}`;
```

Comment:
Is this feature working?
I need to put some comments into a column to let know others devs of a team a small description of what's the purpose of the column.

Reply:
@normancarcamo It was merged into master but you'll have to see which version of sequelize it will land in.

But also this is just the the addition to a PR merged a couple of months back that allowed this to be done on table creation. This PR is just so you could leverage this feature on `addColumn` API of query generator.